ci/packages/components.yml

```diff
@@ -1,4 +1,18 @@
-components build-storybook:
+.run_always_rules: &run_always_rules
```

Comment:
Is `run_always_rules` a good name if it actually does not run always? :) How about just `run_rules`? Or maybe `run_main_branches_only` ?

Reply:
hmm.. let me think

Reply:
maybe `run_all_rules`

Reply:
How about `run_everything_rules`?

Reply:
sounds ok, changed